docs/pages/versions/unversioned/sdk/localization.md

```diff
@@ -64,53 +67,87 @@ i18n.locale = Localization.locale;
 // When a value is missing from a language it'll fallback to another language with the key present.
 i18n.fallbacks = true;
 
-function App() {
+export default App => {
   return (
-    <Text>
-      {i18n.t('welcome')} {i18n.t('name')}
-    </Text>
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {i18n.t('welcome')} {i18n.t('name')}
+      </Text>
+    </View>
   );
-}
+};
+
+/* @hide const styles = StyleSheet.create({ ... }); */
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1,
+  },
+  text: {
+    fontSize: 20,
+  },
+});
+/* @end */
 ```
 
+</SnackInline>
+
 ## API
 
 ```ts
 import * as Localization from 'expo-localization';
 ```
 
-## Constants
-
 ### Behavior
 
 This API is mostly synchronous and driven by constants. On iOS the constants will always be correct, on Android you should check if the locale has updated using `AppState` and `Localization.getLocalizationAsync()`. Initially the constants will be correct on both platforms, but on Android a user can change the language and return, more on this later.
 
+## Constants
+
 ### `Localization.locale`
 
-Native device language, returned in standard format. Ex: `en`, `en-US`, `es-US`.
+An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), consisting of a 2 character language-code and optional script, region and variant codes. Ex: `en`, `en-US`, `zh-Hans`, `zh-Hans-CN`, `en-emodeng`.
 
 ### `Localization.locales`
 
 List of all the native languages provided by the user settings. These are returned in the order the user defines in their native settings.
 
 ### `Localization.region`
 
-**Available on iOS and Web.** Region code for your device which came from Region setting in Language & Region. Ex: US, NZ.
+Region code for your device which came from Region setting in Language & Region. This value is always available on iOS, but might return `null` on Android or web. Ex: `US`, `NZ`, `null`.
```

Comment:
```suggestion
The region code for your device that comes from the Region setting under Language & Region on iOS. This value is always available on iOS, but might return `null` on Android or web. Ex: `US`, `NZ`, `null`.
```

(Where does this code come from on Android and web?)

Reply:
These are obtained from the country/region code in the locale eg. `xx_GB`. And when possible on Android  it is read from the `region` system property, which is available on certain Android devices and was borrowed from react-native-localize.